Directory.Build.targets

```diff
@@ -50,6 +51,9 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsSourceProject)' == 'true'">
+    <None Include="$(AssemblyName).xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
When I added the `<DocumentationFile>` property in the root `Directory.Build.props` but it was not picking up the property for packing. I was getting this error:
```
C:\Users\calope\source\repos\maintenance-packages\.dotnet\sdk\8.0.110\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks 
.Pack.targets(221,5): error NU5026: The file 'C:\Users\calope\source\repos\maintenance-packages\artifacts\bin\System.Net.WebSockets.Web 
SocketProtocol\Release\net462\System.Net.WebSockets.WebSocketProtocol.xml' to be packed was not found on disk. [C:\Users\calope\source\ 
repos\maintenance-packages\src\System.Net.WebSockets.WebSocketProtocol\src\System.Net.WebSockets.WebSocketProtocol.csproj]
```
I fixed it by making sure the xml was copied to the output folder.